transports/sse/runtime/src/main/java/io/quarkiverse/mcp/server/sse/runtime/StreamableHttpMcpMessageHandler.java

```diff
@@ -77,6 +79,7 @@ public class StreamableHttpMcpMessageHandler extends McpMessageHandler<HttpMcpRe
     private final CurrentIdentityAssociation currentIdentityAssociation;
 
     StreamableHttpMcpMessageHandler(McpServersRuntimeConfig config,
+            VertxHttpConfig httpConfig,
```

Comment:
I'm not sure it's reasonable to depend on the `VertxHttpConfig`. There's no clear opinion whether config mappings are part of the public API, and in this case the package  `io.quarkus.vertx.http.runtime` is IMO not a public API. Recently we tried to avoid this kind of dependencies (e.g. using the `HttpRootPathBuildItem`).

I think that something like `ConfigProvider.getConfig().getOptionalValue("quarkus.http.cors.enabled", Boolean.class)` might be more robust. WDYT?

Reply:
HI Martin, done, I can update it further if you'd like, np at all